src/screens/stationSelect.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Text, Button } from 'react-native-paper';
+import DropDownComponent from '../components/dropDown';
+import { ScreenNavigationProps } from '../routes';
+import { DropdownItem } from '../dropDownTypes';
+import { urlOpenerFunc } from '../helpers/urlOpener';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#eee',
+    alignItems: 'center',
+  },
+  dropdownContainer: {
+    flex: 1,
+    backgroundColor: '#eee',
+    alignItems: 'center',
+    margin: 10,
+  },
+  dropdown: {
+    margin: -10,
+  },
+  text: {
+    paddingBottom: 24,
+  },
+});
+
+type StationSelectProps = ScreenNavigationProps<'StationSelect'>;
+
+const stList: DropdownItem[] = [
+  {
+    label: 'Kings Cross',
+    value: 'KGX',
+  },
+  {
+    label: 'Euston',
+    value: 'EUS',
+  },
+  {
+    label: 'Canley',
+    value: 'CNL',
+  },
+  {
+    label: 'Newcastle',
+    value: 'NCL',
+  },
+  {
+    label: 'Orpington',
+    value: 'ORP',
+  },
+];
+
+const StationSelectScreen: React.FC<StationSelectProps> = ({ navigation }) => {
+  const [departStation, setDepartStation] = useState('');
+  const [arriveStation, setArriveStation] = useState('');
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>Select Stations</Text>
+      <DropDownComponent
+        items={stList}
+        label={'Departure Station'}
+        value={departStation}
+        setValue={setDepartStation}
+      />
+      <DropDownComponent
+        items={stList}
+        label={'Arrival Station'}
+        value={arriveStation}
+        setValue={setArriveStation}
+      />
+      <Button
+        mode="contained"
+        onPress={() => {
+          if (
+            departStation &&
```

Comment:
Might be nice to pull this out into a method, e.g.

```typescript
const stationSelectionIsValid: boolean = (departureStation: string | null, arrivalStation: string | null) => {
    // logic here
});
```

It would make it easier to add any more rules in future if the definition of 'valid stations' changes and also is a bit more self-documenting